RecoVertex/PrimaryVertexProducer/src/VertexTimeAlgorithmFromTracksPID.cc

```diff
@@ -0,0 +1,193 @@
+#include "FWCore/Framework/interface/ConsumesCollector.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ParameterSet/interface/ValidatedPluginMacros.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
+#include "vdt/vdtMath.h"
+
+#include "RecoVertex/PrimaryVertexProducer/interface/VertexTimeAlgorithmFromTracksPID.h"
+
+#ifdef PVTX_DEBUG
+#define LOG edm::LogPrint("VertexTimeAlgorithmFromTracksPID")
+#else
+#define LOG LogDebug("VertexTimeAlgorithmFromTracksPID")
+#endif
+
+VertexTimeAlgorithmFromTracksPID::VertexTimeAlgorithmFromTracksPID(edm::ParameterSet const& iConfig,
+                                                                   edm::ConsumesCollector& iCC)
+    : VertexTimeAlgorithmBase(iConfig, iCC),
+      trackMTDTimeToken_(iCC.consumes(iConfig.getParameter<edm::InputTag>("trackMTDTimeVMapTag"))),
+      trackMTDTimeErrorToken_(iCC.consumes(iConfig.getParameter<edm::InputTag>("trackMTDTimeErrorVMapTag"))),
+      trackMTDTimeQualityToken_(iCC.consumes(iConfig.getParameter<edm::InputTag>("trackMTDTimeQualityVMapTag"))),
+      trackMTDTofPiToken_(iCC.consumes(iConfig.getParameter<edm::InputTag>("trackMTDTofPiVMapTag"))),
+      trackMTDTofKToken_(iCC.consumes(iConfig.getParameter<edm::InputTag>("trackMTDTofKVMapTag"))),
+      trackMTDTofPToken_(iCC.consumes(iConfig.getParameter<edm::InputTag>("trackMTDTofPVMapTag"))),
+      minTrackVtxWeight_(iConfig.getParameter<double>("minTrackVtxWeight")),
+      minTrackTimeQuality_(iConfig.getParameter<double>("minTrackTimeQuality")),
+      probPion_(iConfig.getParameter<double>("probPion")),
+      probKaon_(iConfig.getParameter<double>("probKaon")),
+      probProton_(iConfig.getParameter<double>("probProton")),
+      Tstart_(iConfig.getParameter<double>("Tstart")),
+      coolingFactor_(iConfig.getParameter<double>("coolingFactor")) {}
+
+void VertexTimeAlgorithmFromTracksPID::fillPSetDescription(edm::ParameterSetDescription& iDesc) {
+  VertexTimeAlgorithmBase::fillPSetDescription(iDesc);
+
+  iDesc.add<edm::InputTag>("trackMTDTimeVMapTag", edm::InputTag("trackExtenderWithMTD:generalTracktmtd"))
+      ->setComment("");
```

Comment:
```suggestion
  iDesc.add<edm::InputTag>("trackMTDTimeVMapTag", {"trackExtenderWithMTD:generalTracktmtd"});
```
why is the empty `setComment` call neeed?